ui/app/src/components/root/LeftNav.tsx

```diff
@@ -16,7 +16,7 @@ const navLinkGroups: INavLinkGroup[] = [
         url: '/shared-services',
         key: 'shared-services',
         icon: 'Puzzle'
-      },
```

Comment:
nit: Should we just delete this?

Reply:
Gonna leave them in for now as the next stage will bring them back, and I just want the icon names still :)